metaci/repository/migrations/0009_default_implementation_steps.py

```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="repository",
             name="default_implementation_steps",
-            field=models.JSONField(default=[], null=True),
+            field=models.JSONField(default=list, null=True),
```

Comment:
Why are you modifying an existing migration that is not related to the changes in this PR?